MenMa WatchKit Extension/InterfaceController.swift

```diff
@@ -58,11 +58,13 @@ class InterfaceController: WKInterfaceController, WCSessionDelegate {
         
         let messageToSend = ["value": "message sent successfully"]
         session.sendMessage(messageToSend, replyHandler: { replyMessage in
-            //            //handle and present the message on screen
-            //            let value = replyMessage["value"] as? Venue
-            //            //            self.messageLabel.setText(value)
+            //handle and present the message on screen
+            let value = replyMessage["value"] as? Venue
```

Comment:
@lynneokada when venues is passed from the iOS to the watch, it needs to be encoded, and sent by session.sendMessageData method. According to the documentation, and if I understand correctly, replyMessage is 

> A dictionary of property list values containing the response from the counterpart.

 Property list can take only basic type objects: NSString, NSArray, NSDictionary, NSDate, NSNumber. That's why Venue type object can't be stored in replyMessage.


[Link to doc about property list](https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/PropertyLists/AboutPropertyLists/AboutPropertyLists.html)
[Link to doc about replyMessage](https://developer.apple.com/library/prerelease/watchos/documentation/WatchConnectivity/Reference/WCSession_class/index.html#//apple_ref/occ/instm/WCSession/sendMessage:replyHandler:errorHandler:)